packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -7568,7 +7568,13 @@ class Semantics extends SingleChildRenderObjectWidget {
     final bool containsText =
         properties.attributedLabel != null ||
         properties.label != null ||
+        properties.attributedDecreasedValue != null ||
+        properties.attributedIncreasedValue != null ||
+        properties.attributedValue != null ||
+        properties.decreasedValue != null ||
+        properties.increasedValue != null ||
         properties.value != null ||
+        properties.attributedHint != null ||
         properties.hint != null ||
         properties.tooltip != null;
```

Comment:
I'm thinking about how to help make this more easily kept in sync in the future. I think a useful tactic would be to list these attributes in the same order they're declared on SemanticsProperties:

![image](https://github.com/user-attachments/assets/1dfb59c5-c6a5-4a36-8854-c2930515a8e9)

That'd help make it easy to compare this code to that list by eye and check if anything's missing. Conveniently it's also an order that has a nice internal consistency to it: all in pairs `foo` then `attributedFoo` (except the one outlier `tooltip`).

Reply:
@gnprice done in f9ef22a68bdde5dba0c83ca08ee1d5411c0b1470